### PR TITLE
Add Ansible Galaxy badge

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,4 @@ monitoring_nexporter_image_version: latest
 ## Security
 
 This role exposes through an HTTP server lots of data that can be potentially exploited. By default, it listens to a loopback adress, not public from the internet. However, you probably want an external Prometheus server to fetch this data periodically. To this end, and to protect the data, some sort of authentication from the Prometheus server against the host is needed.
-One approach is to leave the exporters binding to localhost and then set up a reverse proxy before them with Basic Authentication, using Nginx. The management of this set up and of the keys implied are out of the scope of this role.
\ No newline at end of file
+One approach is to leave the exporters binding to localhost and then set up a reverse proxy before them with Basic Authentication, using Nginx. The management of this set up and of the keys implied are out of the scope of this role.
```
Comment (enricostano) [inline README.md]: Not sure about where this comes from :thinking:

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Monitoring Role
+# Monitoring Role ![Ansible Role](https://img.shields.io/ansible/role/44192)
```
Comment (enricostano) [inline README.md]: Would be nice to point the URL to https://galaxy.ansible.com/coopdevs/monitoring_role

File: README.md (hunk 1 above)
Comment (raneq) [reply README.md]: What do you mean? I added this section to say "hey, bear in mind that this is not gonna work out of the box" and "one way to complement this role is this".

Do you prefer another example? Not to explain any example?

Comment (enricostano) [reply README.md]: I mean... I didn't touch these lines... and I see no DIFF...

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Monitoring Role ![Ansible Role](https://img.shields.io/ansible/role/44192)
+# Monitoring Role [![Ansible Role](https://img.shields.io/ansible/role/44192)](https://galaxy.ansible.com/coopdevs/monitoring_role)
```
Comment (enricostano) [inline README.md]: :heart_eyes:

**Comment by raneq** [comment]
I see... I did some things to amend the description of the last commit, I'm sure it has to do with that. When I squashed commits in gitlab, the commit title was something like "add molecule" because I initially added it. However, the squashed commit only added node exporter. This is why I tried to amend the commit before the merge commit; I didn't completely succeed, and in the tree remained some void [merge] commits.

I propose you to ignore it. The alternative for me is to start the repo again and commit already with this version.

I'm sorry I messed up.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**